node/packages/sdk/src/webHost.ts

```diff
@@ -26,33 +26,36 @@ export class WebHost {
         options: { baseDir?: string, baseUrl?: string, port?: string | number }
             = { baseDir: '.', baseUrl: null, port: null }
```

Comment:
>baseDir: '.' [](start = 16, length = 12)

maybe this is meant to be baseurl: null? that way it would resolvepath to ../public right below, if nothing is passed in?

Reply:
(Either that, or the samples repo's server.ts files need to be updated, I think)

Reply:
Oh good catch, thanks.